main.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link href="fonts.css" rel='stylesheet' type='text/css'>
+    <title>Save the planet</title>
+</head>
+<body class="body">
+    <header class="nav">
+        <div class="nav__container">
+            <div class="nav__logo">
+                <a class="logo" href="#">
+                    <img class="logo__image" src="Logo.png" alt="Save the planet Logo">
+                </a>
+            </div>
+            <nav class="navbar">
+                <div class="navbar__hamburger">
+                    <span class="navbar__hamburger-bar"></span>
+                </div>
+                <ul class="nav__list">
+                    <div class="nav__logo--dropdown">
+                        <a class="logo--dropdown" href="#">

```

Comment:
daca ai pus variatia pe parinte, poti schimba clasele in interiorul variatiei, nu trebuie sa faci variatie pentru fiecare copil